documentation/modules/oauth/proc-oauth-authentication-broker-config.adoc

```diff
@@ -146,8 +148,10 @@ For production, always use `https://` urls.
 <12> A `scope` parameter passed to the token endpoint. A _scope_ is used when obtaining an access token for inter-broker authentication. It is also used in the name of a client for OAuth 2.0 over PLAIN client authentication using a `clientId` and `secret`. This only affects the ability to obtain the token, and the content of the token, depending on the authorization server. It does not affect token validation rules by the listener.
 <13> The connect timeout in seconds when connecting to the authorization server. The default value is 60.
 <14> The read timeout in seconds when connecting to the authorization server. The default value is 60.
-<15> A JsonPath query used to extract groups information from JWT token or introspection endpoint response. Not set by default. This can be used by a custom authorizer to make authorization decisions based on user groups.
-<16> A delimiter used to parse groups information when returned as a single delimited string. The default value is ',' (comma).
+<15> The maximum number of retries to perform if the request to the authorization server fails in unexpected way. The default value is 0 (no retries). When using this option consider reducing the timeout times for `connectTimeoutSeconds` and `readTimeoutSeconds`. Note that retries prevent the current worker thread from being available to other requests. Many of them stalling could make the Kafka broker unresponsive.
+<16> The time to pause before attempting another retry. By default this time is zero (no pause) as many issues are per-request network glitches or proxy glitches, but if your authorization server is under stress you may want to set this to 100 ms or more.
```

Comment:
```suggestion
<16> The time to wait before attempting another retry of a failed HTTP request to the authorization server. By default, this time is set to zero, meaning that no pause is applied. This is because many issues that cause failed requests are per-request network glitches or proxy issues that can be resolved quickly. However, if your authorization server is under stress or experiencing high traffic, you may want to set this option to a value of 100 ms or more to reduce the load on the server and increase the likelihood of successful retries.
```